subsys/mgmt/mcumgr/smp_bt.c

```diff
@@ -191,10 +305,70 @@ int smp_bt_unregister(void)
 	return bt_gatt_service_unregister(&smp_bt_svc);
 }
 
+/* BT connected callback. */
+static void connected(struct bt_conn *conn, uint8_t err)
+{
+	if (err == 0) {
+		alloc_conn_param_data(conn);
+	}
+}
+
+/* BT disconnected callback. */
+static void disconnected(struct bt_conn *conn, uint8_t reason)
+{
+	struct conn_param_data *cpd = get_conn_param_data(conn);
+
+	/* Cancel work if ongoing. */
+	(void)k_work_cancel_delayable(&cpd->dwork);
+
+	/* Clear cpd. */
+	cpd->default_latency = 0;
+	cpd->latency_state = 0;
+	cpd->conn = NULL;
+}
+
+/* BT LE connection parameters updated callback. */
+static void le_param_updated(struct bt_conn *conn, uint16_t interval,
+			     uint16_t latency, uint16_t timeout)
+{
+	struct conn_param_data *cpd = get_conn_param_data(conn);
+
+	if (latency == 0) {
+		if (cpd->latency_state & CONN_LOW_LATENCY_REQUESTED) {
+			cpd->latency_state &= ~CONN_LOW_LATENCY_REQUESTED;
+			(void)k_work_reschedule(&cpd->dwork, K_MSEC(RESTORE_TIME));
+		}
+		cpd->latency_state = CONN_LOW_LATENCY_ENABLED;
```

Comment:
You never use it anywhere, it's just set here. Also it's not cleared under else here

Reply:
I do clear this, but not explicitly: `cpd->latency_state &= ~CONN_LOW_LATENCY_ENABLED;`
`cpd->latency_state = 0;`